public/js/reducers/user.js

```diff
@@ -4,19 +4,19 @@ const defaults = {
   signedIn: false,
   email: null,
   payment_methods: [],
+  braintreeToken: null,
 };
 
 export default function user(state, action) {
+  console.log('user store: got action', action);
```

Comment:
the reason I haven't been logging at the top level is that a reducer gets called for every action. In other words, this will be noisy because it will include ignored actions. With that said, adding a log statement to every if statement is also maybe a little overkill. It has helped me in the past though.